modules/apps/jaggery/emm/src/emm/units/role-listing/public/js/role-listing.js

```diff
@@ -156,7 +156,7 @@ $("#role-grid").on("click", ".remove-role-link", function () {
 
 $("#search-btn").click(function () {
     var searchQuery = $("#search-by-name").val();
-    if (!searchQuery.trim() == "") {
+    if (searchQuery.trim() != "") {
```

Comment:
var searchQuery = $("#search-by-name").val(); 
if (searchQuery.trim() != "")
is the same logic.but we send the extra spaces since search supports spaces.
specially stop searching to multiple space characters by the filtering here. 
and if we did this search Query dose not support spaces.